firebase-firestore/src/roboUtil/java/com/google/firebase/firestore/ValueHelper.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.firestore;
+
+import com.google.firebase.Timestamp;
+import com.google.firebase.firestore.model.DatabaseId;
+import com.google.firebase.firestore.model.DocumentKey;
+import com.google.firestore.v1.ArrayValue;
+import com.google.firestore.v1.Value;
+import com.google.protobuf.NullValue;
+import com.google.type.LatLng;
+import java.util.List;
+
+/** Test helper to create Firestore Value protos from Java types. */
+public class ValueHelper {
```

Comment:
I'm not a huge fan of classes named with "Util" or "Helper". How about just `Values`? This is a convention found in Guava, where e.g. helpers for creating/manipulating lists are in `Lists`.

Reply:
ValueHelperUtil it is :)

I renamed it to Values. FWIW, in most APIs that use this convention, the helper library and the main library sit in the same package, which we can't do here.

Reply:
`Lists` is a great example because it's a collection of Guava utilities about `java.util.List`. In other cases where the library has a class and static utilities for dealing with it, it makes sense that they'd colocate them. I don't think that invalidates the static utilities as plurals convention.

See go/java-practices/utility-classes.md.